app/views/events/index.html.haml

```diff
@@ -1,15 +1,17 @@
 %section
   .page-header
     .pull-right
-      - if policy(@conference).manage?
+      - if policy(@conference).manage? and !showing_my_events
         = action_button "primary", t('events_module.add_event'), new_event_path, hint: t('events_module.add_event_hint')
         = action_button "", t('events_module.print_cards'), cards_events_path(format: :pdf), hint: t('events_module.print_cards_hint')
         = action_button "", t('events_module.print_accepted_cards') , cards_events_path(accepted: true, format: :pdf), hint: t('events_module.print_accepted_cards_hint')
-    %h1= t('titles.event_list')
+
+    %h1=t('titles.event_list') unless showing_my_events
+    %h1=t('events_module.my_events') if showing_my_events
 
   %ul.tabs
```

Comment:
I wonder if we can also turn those into includes as with the _events_tabs.html.haml … I am asking, because ratings.html.haml does not have the events_path(events: :my) but my_events_path

Reply:
I pushed a new version which combines all the tabs to a separate file. https://github.com/frab/frab/blob/5cb9f31ea56230550f5ff2417537f570862fc2db/app/views/events/_event_list_tabs.html.haml